tests/callbacks/test_finetunning_callback.py

```diff
@@ -0,0 +1,64 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+from torch import nn
+
+from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning.callbacks import BackboneLambdaFinetunningCallback
+from tests.base import BoringModel
+
+
+def test_finetunning_callback(tmpdir):
+    """Test finetunning callbacks works as expected"""
+
+    seed_everything(42)
+
+    class FinetunningBoringModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+            self.backbone = nn.Sequential(nn.Linear(32, 32, bias=False), nn.BatchNorm1d(32), nn.ReLU())
```

Comment:
does it work without a backbone, with pure `BoringModel`?

Reply:
No, it expects a module `backbone`.